src/qemu.rs

```diff
@@ -241,6 +242,24 @@ fn guest_agent_args(sock: &Path) -> Vec<OsString> {
     args
 }
 
+/// Generate general arguments necessary for working with `virtiofs`.
+fn virtiofs_general_args(vm: &VMConfig) -> Vec<OsString> {
+    let mut args: Vec<OsString> = Vec::new();
+
+    args.push("-object".into());
+    args.push(
+        format!(
+            "memory-backend-memfd,id=mem,share=on,size={}",
+            vm.memory.as_str()
```

Comment:
I was a bit confused why `vm.memory` is going to be passed twice, but from docs it seems like this is correct. Maybe call out that we are forcing use of memory sharable w/ virtiofs

Reply:
I added a comment. Let me know if you had something else in mind.